dts/bindings/ethernet/xlnx,axi-ethernet-base.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xilinx AXI 1G/2.5G Ethernet Subsystem base bindings.
+
+include: [base.yaml]
+
+# multiple "compatible" properties match the same driver and options
+
+properties:
+  interrupts:
+    type: array
+    description: Xilinx AXI Ethernet has an optional interrupt for updating statistics.
+  interrupt-parent:
+    type: phandle
+    description: Interrupt controller that the DMA is connected to.
+  reg:
+    type: array
+    description: |
```

Comment:
I expect | here is just keep formatting as it is but here there is no reason for it.

Reply:
Removed